src/screens/SurveyDetail/detail.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+
+import { useHistory } from 'react-router-dom';
+
+import { DetailsContext } from 'contexts/details';
+
+const Detail = () => {
+  const detailsContext = useContext(DetailsContext);
+  const history = useHistory();
+  const goBack = () => history.goBack();
```

Comment:
@rafayet-monon Is the requirement need us to use `history.goBack()`?
I think we can just redirect to the root page when we click on the back button.
Could you give me more clues for this one?
Because for now, I don't see any benefit of that 🤔